google/cloud/pubsub_v1/subscriber/message.py

```diff
@@ -242,9 +243,44 @@ def ack(self) -> None:
                 byte_size=self.size,
                 time_to_ack=time_to_ack,
                 ordering_key=self.ordering_key,
+                future=None
             )
         )
 
+    def ack_with_response(self) -> "pubsub_v1.subscriber.futures.Future":
```

Comment:
As discussed offline, can we add a code sample using:

- ack_with_response
- nack_with_response
- modify_ack_deadline_with_response
- min_duration_per_lease_extension